### PR TITLE
More resource updates for 16.2

### PR DESCRIPTION
**Pull request by tas50**
Bring in a few updates for 16.2 and cleanup formatting.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: content/resources/cron/_index.md [hunk 1]
```diff
@@ -82,11 +86,9 @@ properties_list:
...
   required: false
   description_list:
-  - markdown: 'A Hash of environment variables in the form of
-
-      `({''ENV_VARIABLE'' => ''VALUE''})`. (These variables must exist for a
-
-      command to be run successfully.)'
+  - markdown: 'A Hash containing additional arbitrary environment variables under
+      which the cron job will be run in the form of `({''ENV_VARIABLE'' => ''VALUE''})`.
+      *Note*: These variables must exist for a command to be run successfully.'
```
Comment (IanMadd) [inline content/resources/cron/_index.md:91]: Maybe convert this to a note:
```suggestion
  - note:
    - markdown: These variables must exist for a command to be run successfully.
```

File: content/resources/cron_d/_index.md [hunk 2]
```diff
@@ -111,17 +105,13 @@ properties_list:
...
   description_list:
   - markdown: 'A Hash containing additional arbitrary environment variables under
-
-      which the cron job will be run in the form of
-
-      `({''ENV_VARIABLE'' => ''VALUE''})`. (These variables must exist for a
-
-      command to be run successfully.)'
+      which the cron job will be run in the form of `({''ENV_VARIABLE'' => ''VALUE''})`.
+      **Note**: These variables must exist for a command to be run successfully.'
```
Comment (IanMadd) [inline content/resources/cron_d/_index.md:109]: same thing here:
```suggestion
  - note:
    - markdown: These variables must exist for a command to be run successfully.
```

File: content/resources/cron/_index.md [hunk 3]
```diff
@@ -82,11 +86,9 @@ properties_list:
   ruby_type: Hash
   required: false
   description_list:
-  - markdown: 'A Hash of environment variables in the form of
-
-      `({''ENV_VARIABLE'' => ''VALUE''})`. (These variables must exist for a
-
-      command to be run successfully.)'
+  - markdown: 'A Hash containing additional arbitrary environment variables under
+      which the cron job will be run in the form of `({''ENV_VARIABLE'' => ''VALUE''})`.
```
Comment (IanMadd) [inline content/resources/cron/_index.md:90]: If you change the text below to a note then I think you have to add `'` to the end of this string:
```suggestion
  - markdown: 'A Hash containing additional arbitrary environment variables under
      which the cron job will be run in the form of `({''ENV_VARIABLE'' => ''VALUE''})`.'
```

File: content/resources/cron_d/_index.md [hunk 4]
```diff
@@ -111,17 +105,13 @@ properties_list:
   required: false
   description_list:
   - markdown: 'A Hash containing additional arbitrary environment variables under
-
-      which the cron job will be run in the form of
-
-      `({''ENV_VARIABLE'' => ''VALUE''})`. (These variables must exist for a
-
-      command to be run successfully.)'
+      which the cron job will be run in the form of `({''ENV_VARIABLE'' => ''VALUE''})`.
```
Comment (IanMadd) [inline content/resources/cron_d/_index.md:108]: If you change the text below to a note then I think you have to add the `'` to the end of this string.
```suggestion
      which the cron job will be run in the form of `({''ENV_VARIABLE'' => ''VALUE''})`.'
```

**Review by IanMadd — COMMENTED** [review]
Maybe convert the `*Note*:` to a proper note for the cron and cron_d resources

File: content/resources/cron/_index.md (hunk 1 above)
Comment (tas50) [reply content/resources/cron/_index.md:91]: I was trying to avoid the full on notes since they're pretty intrusive when reading over the properties.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**